web/src/components/faq.jsx

```diff
@@ -86,24 +101,30 @@ const FAQ = ({ className }) => {
   }, 50);
 
   return (
-    <div className={className} ref={ref}>
-      <div className="faq-container">
-        {orderings.map(({ groupKey, entryOrder }) => (
-          <div className="question-group-container" key={groupKey}>
-            <div className="question-group-header title">{__(`${groupKey}.groupName`)}</div>
-            {entryOrder.map((entryKey) => (
-              <QuestionAnswer
-                key={entryKey}
-                groupKey={groupKey}
-                entryKey={entryKey}
-                setAnswerVisible={setAnswerVisible}
-                answerVisible={answersVisible[entryKey]}
-              />
-            ))}
-          </div>
-        ))}
+    <React.Fragment>
+      <div className={className} ref={ref}>
+        <div className="faq-container">
+          {orderings.map(({ groupKey, entryOrder }) => (
+            <div className="question-group-container" key={groupKey}>
+              <div className="question-group-header title">{__(`${groupKey}.groupName`)}</div>
+              {entryOrder.map((entryKey) => (
+                <QuestionAnswer
+                  key={entryKey}
+                  groupKey={groupKey}
+                  entryKey={entryKey}
+                  setAnswerVisible={setAnswerVisible}
+                  answerVisible={answersVisible[entryKey]}
+                />
+              ))}
+            </div>
+          ))}
+        </div>
       </div>
-    </div>
+      <TermsAndPrivacyContainer>
+        <a href="https://www.electricitymap.org/privacy-policy/">{__('misc.privacyPolicy')}</a>
+        <a href="https://www.electricitymap.org/legal-notice/">{__('misc.legalNotice')}</a>
```

Comment:
Would it make sense to add icons like these here?
![image](https://user-images.githubusercontent.com/30777521/171857192-41867b35-3012-4205-bcc8-f6818a55971b.png)